uPortal-webapp/src/main/webapp/media/skins/common/javascript/uportal/up-ga.js

```diff
@@ -45,6 +45,21 @@ var uportal = uportal || {};
         return up.analytics.model.defaultConfig;
     };
 
+    /**
+     * Set the defaultConfig.config array as global settings
+     */
+    var configureDefaults = function (propertyConfig) {
+        //console.log(propertyConfig);
```

Comment:
Do we still need the commented out logs?
Could we remove them?

Reply:
Great question and one I anticipated. I think some of the clients will need to leverage the console logs to conform their configuration is correct. I can go either way. What do you think?